modules/olm-defining-csv-webhooks.adoc

```diff
@@ -1,57 +1,118 @@
 // Module included in the following assemblies:
 //
-// * operators/olm-webhooks.adoc
+// * operators/operator_sdk/osdk-generating-csvs.adoc
 
 [id="olm-defining-csv-webhook_{context}"]
-= Defining webhooks in a CSV
+= Defining webhooks
 
-The ClusterServiceVersion (CSV) resource includes a `webhookdefinitions` section
-to define validating and mutating admission webhooks that ship with an Operator.
-For example:
+Webhooks allow Operator authors to intercept, modify, and accept or reject
+resources before they are saved to the object store and handled by the Operator
+controller. Operator Lifecycle Manager (OLM) can manage the lifecycle of these
+webhooks when they are shipped alongside your Operator.
 
-.CSV containing a validating admission webhook
+The ClusterServiceVersion (CSV) resource of an Operator can include a
+`webhookdefinitions` section to define the following types of webhooks:
+
+* Admission webhooks (validating and mutating)
+* Conversion webhooks
+
+.Procedure
+
+* Add a `webhookdefinitions` section to the `spec` section of the CSV of your
```

Comment:
Are we assuming the user knows where the CSV is located and how to access it? 
More so a question for my understanding :) I'm fine with it as written.

Reply:
At this point in the overall http://file.rdu.redhat.com/~adellape/100620/conversion_webhooks/operators/operator_sdk/osdk-generating-csvs.html assembly, yeah, they should know where they've stored their Operator project's CSV.